valhalla/baldr/pathlocation.h

```diff
@@ -35,7 +35,8 @@ struct PathLocation : public Location {
              const double score,
              const SideOfStreet sos = NONE,
              const unsigned int outbound_reach = 0,
-             const unsigned int inbound_reach = 0);
+             const unsigned int inbound_reach = 0,
+             const float projected_heading = 0.f);
```

Comment:
this should probably be some sentinal value that indicates not valid. `-1.f` for example.

Reply:
the other option is to change the order of the parameters and make it a manditory parameter. although that is more code change its more of the kind of change we are making here. basically every path edge must have a heading. another problem with that one though is that there may be several callers outside of loki::search that dont know or care about the heading.

Reply:
I made it mandatory, there were hardly any other callers other than in tests